src/components/trashlookup.jsx

```diff
@@ -1,156 +1,121 @@
+          .add(6 - today, 'd')
+          .format(dateFormat)
+      : today === dayOfWeek.saturday
+      ? moment()
+          .add(2, 'd')
+          .format(dateFormat)
+      : moment()
+          .add(1, 'd')
+          .format(dateFormat);
+  }
 
-    const searchQuery = this.state.addresses;
-    const searchValue = this.state.searchQuery;
+  render() {
+    const { data } = this.addressData();
+    const isCollectionScheduleTextHidden = this.state.isCollectionScheduleTextHidden;
+    const isAutoTextHidden = this.state.isAutoTextHidden;
     return (
       <React.Fragment>
-        <p>Showing {totalCount} address in the database.</p>
-        <SearchBox
-          value={searchValue}
-          onChange={this.handleSearch}
-          onClick={this.handleSearchClick}
-        />
+        <h6>Find Your Collection Schedule.</h6>
+        <p>First paragraph shown here</p>
+        <p>Second paragraph shown here</p>
         <div className="row">
-          {issearch === 1 && totalCount > 0 ? (
-            <div className="col-5">
-              <ListGroup
-                items={data}
-                selectedItem={this.state.selectedAddress}
-                onItemSelect={this.handleAddressSelect}
-              />
-            </div>
-          ) : (
-            ""
-          )}
+          <div className="col-5">{!isAutoTextHidden && this.renderList(data)}</div>
         </div>
-        {issearch === 3 ? (
-          <table className="table">
-            <thead>
-              <tr>
-                <th>Type</th>
-                <th>Image</th>
-                <th>Collection Days</th>
-                <th>Next Collection</th>
 
-                <th />
-              </tr>
-            </thead>
-            <tbody>
-              {this.state.services.map(service => (
-                <tr key={service._id}>
-                  <td>{service.type}</td>
-                  <td>
-                    {service.type === "Trash" ? (
-                      <Trash />
-                    ) : service.type === "Leaf" ? (
-                      <Leaf />
-                    ) : (
-                      <Recycle />
-                    )}
-                  </td>
-                  <td>{service.collectionDays}</td>
-                  <td>{service.nextCollection}</td>
-                </tr>
-              ))}
-            </tbody>
-          </table>
-        ) : issearch === 2 && totalCount > 0 ? (
-          <div>
-            we could not find {searchValue}
-            <p>Did you mean?</p>
-            {data[0].address1}
-          </div>
-        ) : (
-          ""
-        )}
+        {!isCollectionScheduleTextHidden && this.displayCollectionSechduleText()}
 
-        {/* {data.length === 1 ? data[0].address1 : "none"}
-        <ul className="list-group-item">
-          {data.map((item, i) => (
-            <div key={i}> {item.address1}</div>
-          ))}
-        </ul> */}
+        {!isCollectionScheduleTextHidden && this.trashSchedule()}
```

Comment:
in most cases where you have methods that start with "render" should be moved to their own components. If you need something from this component they should be passed as props.

Reply:
created TrashSchedule component